examples/vision/attention_mll_classification.py

```diff
@@ -0,0 +1,541 @@
+"""
+Title: Classification using Attention-based Deep MIL
+Author: [Mohamad Jaber](https://www.linkedin.com/in/mohamadjaber1/)
+Date created: 2021/08/16
+Last modified: 2021/08/16
+Description: A neural network model that uses attention weights on instance-level to decide the output class.
+"""
+"""
+## Introduction
+
+Instead of having individual labelled instances, this neural network model receives a single label for each bag of instances.
```

Comment:
Need some high-level intro of the problem first: what is it and why is it useful? Explain it like you would explain to highschooler

Reply:
I would provide an introduction that does the following:
1. Explain what MIL is, as well as its applications
2. Describe the Attention-based Deep MIL model

Reply:
I begin now my example by giving a simple common used example for MIL. Then gradually start elaborating on Attention-based Deep MIL model, its need and how it could be useful.